docs/src/pages/components/cards/CustomizedCard.js

```diff
@@ -0,0 +1,173 @@
+  main: '#007FFF',
+  500: '#007FFF',
+  600: '#0072E5', // vs blueDark 900: WCAG 4.6 AAA (large), APCA 36 Not for reading text
+  700: '#0059B2',
+  800: '#004C99',
+  900: '#003A75',
+};
+
+const theme = createTheme({
+  palette: { mode: 'dark', primary },
+  shape: {
+    borderRadius: 10,
+  },
+  spacing: 10,
+  typography: {
+    fontFamily: [
+      'IBM Plex Sans',
+      '-apple-system',
+      'BlinkMacSystemFont',
+      'sans-serif',
+    ].join(','),
+    h6: {
+      lineHeight: 1.2,
+    },
+  },
+  components: {
+    MuiPaper: {
+      variants: [
+        {
+          props: { variant: 'gradient' },
+          style: {
+            background: `linear-gradient(to right bottom, ${primary.main}, ${primary[700]} 120%)`,
+            boxShadow:
+              '0px 20px 25px rgba(0, 0, 0, 0.1), 0px 10px 10px rgba(0, 0, 0, 0.04)',
+          },
+        },
+      ],
+    },
+    MuiAvatar: {
+      styleOverrides: {
+        root: {
+          border: '2px solid #fff',
+        },
+      },
+    },
+    MuiLinearProgress: {
+      styleOverrides: {
+        root: {
+          borderRadius: 10,
+          backgroundColor: primary[500],
+        },
+        bar: {
+          borderRadius: 10,
+          backgroundColor: '#fff',
+        },
+      },
+    },
+  },
+});
+
+export default function CustomizedCard() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Fade in timeout={700}>
+        <Card
+          variant="gradient"
+          sx={{
+            minWidth: 280,
+            maxWidth: 360,
+            minHeight: 280,
+            display: 'flex',
+            flexDirection: 'column',
+            p: 2,
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+            }}
+          >
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+              <Typography
+                color="primary.100"
+                variant="caption"
+                fontWeight={500}
+                sx={{ mt: 0.3 }}
+              >
+                March 25th
+              </Typography>
+            </Box>
+            <IconButton size="small">
+              <MoreVertIcon fontSize="small" />
+            </IconButton>
+          </Box>
+          <Box sx={{ my: 'auto' }}>
+            <Box
+              sx={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+                width: 25,
+                height: 25,
+                bgcolor: '#fff',
+                borderRadius: 0.5,
+                p: '2px',
+              }}
+            >
+              <CodeRounded color="primary" fontSize="small" />
+            </Box>
+            <Typography
+              variant="h6"
+              component="div"
+              sx={{ mt: 1.5, fontWeight: 500 }}
+            >
+              Check the docs for getting every component API
```

Comment:
Rather than fixing the grammar, should we change it to something that makes sense in the context of a "todo" card?

Reply:
Sure, do you have any suggestions?

Reply:
"Develop custom theme for MUI components" 😀